airbyte-webapp/src/views/Connector/ServiceForm/components/Sections/auth/useOauthFlowAdapter.tsx

```diff
@@ -38,13 +46,19 @@ function useFormikOauthAdapter(connector: ConnectorDefinitionSpecification): {
     }
 
     setValues(newValues);
+    setHasRun(true);
   };
 
   const { run, loading, done } = useRunOauthFlow(connector, onDone);
+  const connectionObjectEmpty =
+    values?.connectionConfiguration?.credentials ?? values?.connectionConfiguration
+      ? Object.keys(values.connectionConfiguration?.credentials ?? values.connectionConfiguration).length <= 1
```

Comment:
`values.connectionConfiguration` is the object in the Formik form values that contains all the configuration of the connector, meaning that it may return an incorrect values if the user fills out all the fields except oauth because it would mean that values.connectionConfiguration > 1, therefore not empty?

Reply:
Just pushed a change that catches this. This is such a complex workflow!